cypress/e2e/tests/pages/cluster-manager.spec.ts

```diff
@@ -14,7 +16,9 @@ const runPrefix = `e2e-test-${ runTimestamp }`;
 
 // File specific consts
 const { baseUrl } = Cypress.config();
-const clusterRequestBase = `${ baseUrl }/v1/provisioning.cattle.io.clusters/fleet-default`;
+const namespace = 'fleet-default';
+const type = 'provisioning.cattle.io.cluster';
```

Comment:
There's a diff between the new and old (`s`)

`provisioning.cattle.io.clusters` vs `provisioning.cattle.io.cluster`

Reply:
nice catch, thanks :)